project/security.md

```diff
@@ -37,12 +37,18 @@ innocent and well-meaning but has a disguised malicious intent.
 
 To mitigate such risks, we apply established procedures and techniques:
 
+- **GitHub**. The curl project uses <https://github.com/curl/curl> as its main
+  source code git repository since 2010. We rely on their hosting to secure
+  access as per our configuration.
 - **2FA required**. We require all maintainers with push access to git to have
   two-factor authentication enabled, to reduce the risk that attackers can
-  impersonate them and use their credentials to push source code changes.
+  impersonate them and use their credentials to push source code changes. We
+  rely on GitHub's 2fa setup.
```

Comment:
I think it would make sense to capitalize 2fa/2FA consistently, and in the "heading" it's capitalized.

Reply:
I'll fix that in #501!